src/tests/system/tests/test_kcm.py

```diff
@@ -413,3 +415,335 @@ def test_kcm__simple_kinit(client: Client, kdc: KDC):
             with pytest.raises(SSHProcessError):
                 krb.kinit(username, password="wrong")
             assert krb.klist().rc == 0, "Klist failed"
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_kcm__kinit_kcm_krb5ccname(client: Client, kdc: KDC):
```

Comment:
This test case can be removed. This is a generic kcm kinit test, while several of the tests already perform a kinit.